docs/core_components/pages/streamfield.rst

```diff
@@ -0,0 +1,412 @@
+Freeform page content using StreamField
+=======================================
+
+Wagtail's content model encourages thinking about the information on a page in a formal, structured way; for example, a page about an event is not just an assortment of headings, paragraphs and images, but has specific fields for dates, location, a list of speakers and so on. This ensures that the template developer has a known, predictable set of fields to work with, and can present that information consistently throughout the site in the most appropriate arrangement. It also allows the information to be re-used and repurposed, such as displaying an "events near you" listing.
```

Comment:
I'm reluctantly inclined to second this, despite matthew's eloquent explanation of a difficult concept :P

Some of the background info could probably be removed and the rest abbreviated.